dist/ledger/README-cli.md

```diff
@@ -361,3 +361,139 @@ This concludes the upgrade process. The device is now ready to be used with the
 ## What's next
 
 Once the powHSM device is properly setup and onboarded, it is ready to be used with the powHSM middleware.
+
+## Troubleshooting
+
+This section lists some common issues that might arise during the setup and onboarding
+process and provides guidance on how to solve them.
+
+### Ledger Nano S screen is too dim
+
+Unfortunately, it is a well known problem that after a long time of usage, the
+Nano S screen might start dimming, eventually reaching a point where it is
+nearly impossible to read the on-screen instructions. This is a hardware problem and there are
+some workarounds offered both by [Ledger](https://support.ledger.com/article/360021124674-zd)
+and [third-party websites](https://symetronix.com/ledger-nano-s-screen-not-working-comprehensive-guide-to-fix-the-issue/). 
+In any case, it is still possible to perform the process described in this document,
+even if the screen is completely unreadable. This section lists detailed steps so
+that the onboarding can be performed even without the possibility of reading any
+of the on-screen instructions.
+
+For the update process, there's no need for the user to do anything besides plugging
+and unplugging the device when prompted.
+
+#### Detailed steps for onboarding a new device
+
+This step is needed to onboard a new device. To access the Recovery Mode, follow
+the steps below:
+
+1. Press and hold the Right button on the Nano S. This is the button furthest
+   from the USB port. Keep it pressed while you connect the USB cable to the computer.
+   After connecting the cable, keep the button pressed for another 5 seconds, and then
+   release it. Wait for another 5 seconds before proceeding to the next step.
+
+> Now there are two ways to proceed, depending on whether or not the device has
+  a pin set. Devices that have been onboarded and are currently in use have a pin
+  set, so move on to step 2. Devices that have been onboarded, but have been wiped
+  (i.e., the wrong pin was entered three times in a row) do not have a pin set,
+  so skip to step 3. Brand new devices also do not have a pin set, so skip to step 3.
```

Comment:
Re reading this, I feel it's a bit confusing for the user given that it's written in prose in a single paragraph. Can we make it clearer by writing it in an itemised way in the form of "If A, then go to step N"?